web/src/locales/en.json

```diff
@@ -103,5 +105,15 @@
 	"employees.schedule": "Schedule",
 	"employees.role": "Role",
 	"employees.role.manager": "Manager",
-	"employees.role.wasteOperator": "Waste operator"
+	"employees.role.wasteOperator": "Waste operator",
+	"employees.birthdate": "Birthdate",
+	"employees.phone": "Phone",
+	"employees.location": "Location",
+	"employees.scheduleStart": "Schedule start",
+	"employees.scheduleEnd": "Schedule end",
+	"employees.notFound.title": "Employee not found",
+	"employees.notFound.description": "The employee you are looking for does not exist.",
+	"employees.delete.success": "Employee deleted successfully",
+	"employees.delete.conflict.title": "Conflict",
+	"employees.delete.conflict.description": "This employee cannot be deleted because it's associated with a report or a route."
```

Comment:
@Goncalo-Marques Does it still make sense to include information about reports in the message?

Reply:
I think we can keep it since we already expect to do that as future work.
Also, I think we can keep that information to maintain consistency with the truck conflict message, which indicates that it can be associated with a warehouse, but we also do not expect to implement the truck-warehouse association in the first project iteration.